src/simtools/simtel/simulator_light_emission.py

```diff
@@ -180,6 +297,30 @@ def _write_telpos_file(self):
 
         return telpos_file
 
+    def _prepare_ff_atmosphere_files(self, config_directory: Path) -> int:
+        """Prepare canonical atmosphere aliases for ff-1m and return model id 1."""
+        atmo_name = self._telescope_model.get_parameter_value("atmospheric_profile")
```

Comment:
Is this actually working? `atmospheric_profile` should be a site model parameter, and not a telescope model parameter.